src/stest/src/file.rs

```diff
@@ -175,6 +173,13 @@ impl File {
         self.mode().map(is_executable)
     }
 
+    pub fn file_name(&self) -> String {
+        self.path_buf
+            .file_name()
+            .and_then(|f| Some(f.to_string_lossy().to_string()))
```

Comment:
Is there a reason you use `and_then` rather than just `map`?
```suggestion
            .map(|f| f.to_string_lossy().to_string())
```

Reply:
no reason. updated the pr. thanks for the suggestion.